ubuntu-dev/workspace-janitor.js

```diff
@@ -71,6 +71,21 @@ module.exports = function (options) {
 
           // Use a longer scrollback for the Terminal.
           p.settings.user.terminal['@scrollback'] = 10000;
+
+          if (!p.settings.project) {
+            p.settings.user = {};
```

Comment:
Nit: You probably mean to create `p.settings.project` here, as opposed to `p.settings.user`.

Reply:
fixed